packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -75,6 +75,14 @@ const ASPECT_RATIO_TOLERANCE = .1;
  */
 const OVERSIZED_IMAGE_TOLERANCE = 1000;
 
+/**
+ * Used to limit automatic srcset generation of very large sources for
+ * fixed-size images.
```

Comment:
nit: similar to the `OVERSIZED_IMAGE_TOLERANCE ` above - we can mention that it's in pixels.
```suggestion
 * Used to limit automatic srcset generation of very large sources for
 * fixed-size images. In pixels.
```